introduction.md

```diff
@@ -1 +1,6 @@
-Hello, my name if Jeff and I'm new to the dojo. Javascript is my favorite Code School path.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?